test/kpod.bats

```diff
@@ -7,3 +7,33 @@ load helpers
 	echo "$output"
 	[ "$status" -eq 0 ]
 }
+
+@test "kpod pull from docker with tag" {
+	run ${KPOD_BINARY} pull debian:6.0.10
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
```

Comment:
We should be doing rmi tests as well to keep the database clean. or else these will only test once.